civitai-downloader-v2/src/api/cache.py

```
@@ -1,5 +1,330 @@
+        try:
+            if isinstance(data, (str, bytes)):
+                return len(data)
+            elif isinstance(data, (int, float)):
+                return sys.getsizeof(data)
+            elif isinstance(data, (list, tuple)):
+                return sum(self._estimate_size(item) for item in data)
+            elif isinstance(data, dict):
+                return sum(self._estimate_size(k) + self._estimate_size(v) for k, v in data.items())
+            else:
+                return sys.getsizeof(data)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)